lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -93,8 +114,9 @@ void _testEngineSemanticsOwner() {
     expect(semantics().semanticsEnabled, false);
 
     // Synthesize a click on the placeholder.
+    final root = getRootDocument();
```

Comment:
It's a bug in our analyzer options right now, but we should try to follow the Flutter style guide and specify types everywhere.

Reply:
There's no type that expresses what this is able to do (you can use a document or a shadow root interchangeably), but yeah, I can make this a ShadowRoot :P

Reply:
These are all gone, and used inline instead.